pet.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+from subprocess import *
+import os
+import signal
+import click
+
+
+cli = click.Group()
+
+
+@cli.command()
+@click.argument('name')
+def start(name):
+    """starts new project"""
+    if os.path.exists("./projects/%s" % name):
+        if not os.path.isfile("./projects/%s/_lock" % name):
+            open("./projects/%s/_lock" % name, 'a')
+            Popen(["./start.sh", name, "./projects/%s" % name]).communicate(input)
+        else:
+            click.echo("project already activated")
+    else:
+        click.echo("project not found")
+
+
+@cli.command()
+def stop():
+    """stops project"""
+    os.kill(os.getppid(), signal.SIGKILL)
+
+
+# add working with templates
+@cli.command()
+@click.argument('name')
+@click.option('--temp', '-m', multiple=True, default=None, help='Creates project using templates')
+def create(name, temp):
+    """creates new project"""
+    if not os.path.exists("./projects/%s" % name):
```

Comment:
if it uses templates as `multiple=True` it should probably iterate over those templates?